python/coral_usb/util.py

```diff
@@ -30,6 +30,35 @@ def get_panorama_sliced_image(panorama_img, panorama_slice):
     return img
 
 
+def get_tiles(width, height, overlap=True,
```

Comment:
```suggestion
def get_tiles(img_height, img_width, overlap=True,
```

please use `y, x` order

Reply:
please use `y, x` order. also use `img_height, img_width`.